### ISSUE TITLE
[FEATURE] Notification shouldn't work while the tab is active

### ISSUE THREAD

**Issue by Dun-sin** [Dun-sin/Whisper#366]
### Description

The notification shouldn't work if the app tab is active, here's where the fix should focus on:
https://github.com/Dun-sin/Whisper/blob/1b7473a7b0732352db776c98964868a3338981cb/client/src/lib/browserNotification.js#L10-L18

### 👀 Have you checked if this issue has been raised before?

- [X] I checked and didn't find similar issue

### 🏢 Have you read the Contributing Guidelines?

- [X] I have read and understood the rules in the [Contributing Guidelines](https://github.com/Dun-sin/Whisper/blob/main/CONTRIBUTING.md)

### Are you willing to work on this issue ?

None

**Comment by Virtual4087** [comment]
Can you assign this to me?

**Comment by Dun-sin** [comment]
> Can you assign this to me?

Definitely, thanks for wanting to contribute, Make sure you follow the rules [👉🏽here👈🏽](https://github.com/Dun-sin/Whisper/blob/main/CONTRIBUTING.md#rules), else your PR will not be accepted and will be closed. Good Luck

**Comment by Virtual4087** [comment]
My bad i had pushed the main branch instead of creating another one
